packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1663,8 +1663,6 @@ export class ContainerRuntime
 				!readonly || !this.connected,
 				0x125 /* "Unsafe to transition to read-only state!" */,
 			);
-
-			this.replayPendingStates();
```

Comment:
cc @kian-thompson 😃

Reply:
This old PR might be related: https://github.com/microsoft/FluidFramework/pull/4239